platform/lib/templates/index.js

```diff
@@ -91,50 +86,14 @@ class Templates {
    * it'll load the template from the pages directory.
    */
   async load(templatePath) {
-    return this.compile(templatePath, async () => {
-      if (config.isTestMode()) {
-        // fetch original doc page from filesystem for testing
-        return readFileAsync(join(paths.PAGES_SRC, templatePath), 'utf-8');
-      } else if (config.isDevMode()) {
-        // fetch doc from proxy
-        return this.fetchTemplate_(templatePath);
-      } else {
-        // fetch comiled doc page from filesystem
-        return readFileAsync(pagePath(templatePath), 'utf-8');
-      }
-    });
-  }
-
-  /**
-   * Loads a template from cache (if not in dev mode). If the template
-   * is not cached, it will use the provided callback to retrieve the
-   * template string.
-   */
-  async compile(key, fn) {
```

Comment:
`compile` is still called in 

https://github.com/ampproject/amp.dev/blob/0281a56be55127312ed7af3887f33fcc9e792b2d/platform/lib/templates/index.js#L61

Reply:
Fixed with https://github.com/ampproject/amp.dev/pull/2613